charts/pastebin/ci/test-values.yaml

```diff
@@ -0,0 +1,53 @@
+cert-manager:
+  install: true
+
+configMap:
+  data:
+    DB_HOST: "dpaste-headless"
+    DB_NAME: "dpaste"
+    DB_PASS: "default-non-secure-password"
+    DB_PORT: "3306"
+    DB_USER: "dpaste"
+    SESSION_KEY: "dpaste-session-key"
+    VAL: "5"
+
+deployment:
+  port: 5000
+  replicaCount: 1
+
+imagePullSecrets:
+  - name: ecr-registry
+
+ingress:
+  className: nginx
+  hosts:
+    - host: paste.allizom.com
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+          serviceName: pastebin
+          servicePort: 80
+  labels: {}
+  le:
+    name: stage
+    issuer_create: false
+  name: pastebin
+  tls:
+    - hosts:
+        - paste.allizom.com
```

Comment:
We don't control `allizom.com`; should this be `allizom.org`? (foreach x4 instances of .com in this PR)

Reply:
these are just test values and not used in our actual deployments (see https://github.com/mozilla-it/itse-apps-stage-1-infra/blob/6fe99189a210debfd100998b7f25e0f41041831d/k8s/releases/pastebin/pastebin.yaml#L62 as one example).

will note this for a cleanup item when i come through with etherpad helm chart though.